pkg/plumber/types.go

```diff
@@ -7,76 +7,41 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// PlumberJobType specifies how the job is triggered.
-type PlumberJobType string
+// PipelineKind specifies how the job is triggered.
+type PipelineKind string
 
 // Various job types.
 const (
 	// PresubmitJob means it runs on unmerged PRs.
-	PresubmitJob PlumberJobType = "presubmit"
+	PresubmitJob PipelineKind = "presubmit"
```

Comment:
Shouldn't we change `presubmit`, `postsubmit`, `periodic` and `batch` to jenkins-x pipeline specific values: `pullRequest`, `release`, `feature`. We will map the webhook events directly to these and not rely anymore on prow terminology as a middle layer.

Reply:
I thought we decided for now keep the prow configuration format; so I figured leaving the `presubmit/postsubmit` terms until we revamp the config to minimise confusion.

I figure we can replace the prow config + the presubmit/postsubmit terms at a later date?

Reply:
Agree. Let's keep the existing prow types and names to avoid confusion until we do a more radical improvement in the config.